package.json

```diff
@@ -30,7 +30,7 @@
   },
   "homepage": "https://github.com/xpbrew/cordova-sqlite-storage",
   "dependencies": {
-    "cordova-sqlite-storage-dependencies": "2.0.0"
+    "cordova-sqlite-storage-dependencies": "https://github.com/OutSystems/cordova-sqlite-storage-dependencies#2.0.0-OS1"
```

Comment:
-OS1 suffix 😞

Reply:
Shouldn't it be `+OS1` since it's something we added on top of `2.0.0`?

Reply:
@oscsx that's a good point! We have a task to review the version convention for forks and to normalize all the versions in every forked component that we are using. We can then add you to the loop for more feedback.